react-app/src/components/Grid.tsx

```diff
@@ -0,0 +1,27 @@
+import { CSSProperties, ReactNode } from "react";
+import styles from "../styles/GridPage.module.css";
+
+export default function Grid({
+  cssProps,
+  children,
+}: {
+  cssProps: CSSProperties;
+  children: ReactNode;
+}) {
```

Comment:
props should be imported type

Reply:
I don't know how to use them that way if one of them is children. Maybe I should import them separately?